scripts/makei

```diff
@@ -161,12 +164,27 @@ def add_cvtsrcpf_parser(subparsers: argparse.ArgumentParser):
         help='the name of the library',
         metavar='<library>',
     )
+    cvtsrcpf_parser.add_argument(
+        "--stmfccsid",
```

Comment:
Why would we need to support anything besides 1208?

Reply:
I think that's because some compilers don't support compiling from UTF8. Is that correct @worksofliam ?

Reply:
That's correct.  None of the ILE compilers support it.  But with the TGTCCSID parameter at least the ILE RPG compilers do as of late.  I have to investigate exactly which compilers support it